service/service.go

```diff
@@ -145,16 +173,17 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
-	var newControllerBackOff *backoff.ExponentialBackOff
+	var newBackOff *backoff.ExponentialBackOff
```

Comment:
The framework also requires a backoff now, so let's use the configured one.

Reply:
I have no evidence but I would say the backoff is a pointer and you should never reuse it for different things at the same time. IMO create one backoff for each component that requires one.

Reply:
Yeah, 100%. I'll update. Didn't think about backoff internal state.